go.mod

```diff
@@ -111,7 +116,8 @@ require (
 	github.com/felixge/httpsnoop v1.0.2 // indirect
 	github.com/go-logr/logr v1.2.2 // indirect
 	github.com/go-logr/stdr v1.2.2 // indirect
-	github.com/golang-jwt/jwt/v4 v4.1.0 // indirect
+	github.com/golang-jwt/jwt v3.2.1+incompatible // indirect
+	github.com/golang-jwt/jwt/v4 v4.2.0 // indirect
```

Comment:
Looks like V4 is used by https://github.com/moby/buildkit/blob/170b2b0a5720e17e8972441f782b8ebaaa83ec99/vendor/github.com/tonistiigi/go-actions-cache/cache.go#L22

And v3 by https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/751ec246d696ada22aee7a7c3bfd7a097b248250/apps/internal/oauth/ops/accesstokens/accesstokens.go#L33

I know v3.2.1 has some potential vulnerabilities (see https://github.com/golang-jwt/jwt/pull/40 / https://github.com/golang-jwt/jwt/releases/tag/v3.2.2), and v3 is unmaintained;  do you know if there's plans to upgrade to v4?

Reply:
v0.6.0 of github.com/AzureAD/microsoft-authentication-library-for-go has moved to v4, so will add a replace in the gomod for now. the newer version of github.com/Azure/azure-sdk-for-go/sdk/azcore are scheduled to use that version, so when we upgrade to those we can remove the replace